.github/workflows/auto-start-ci.yml

```diff
@@ -34,16 +33,14 @@ jobs:
 
       # Get Pull Requests
       - name: Get Pull Requests
-        uses: octokit/graphql-action@v2.x
+        uses: octokit/graphql-action@v2
```

Comment:
not sure this will work, as there is no `v2` tag. `v2.x` is a branch: https://github.com/octokit/graphql-action/tree/v2.x

Reply:
No, `2.x` points to a released version (in this case the latest `v2`): https://github.com/marketplace/actions/github-graphql-api-query.

Reply:
My point is: `v2` points to nothing (as far as I can tell).

Reply:
~~It works like `actions/checkout@v2`, for example.~~ (?)
https://docs.github.com/en/actions/creating-actions/about-actions#using-tags-for-release-management
Maybe you are right, I will test this line.